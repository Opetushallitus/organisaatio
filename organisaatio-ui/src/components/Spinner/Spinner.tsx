import React from 'react';
import Spin from '@opetushallitus/virkailija-ui-components/Spin';


const Spinner = () => {
    return (
        <div>
        </div>
    )
}

export default Spinner;