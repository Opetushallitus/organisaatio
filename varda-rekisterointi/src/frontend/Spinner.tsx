import React from 'react';

export default function Spinner() {
    return (
        <div className="oph-spinner">
            <div className="oph-bounce oph-bounce1"></div>
            <div className="oph-bounce oph-bounce2"></div>
            <div className="oph-bounce oph-bounce3"></div>
        </div>
    )
}
