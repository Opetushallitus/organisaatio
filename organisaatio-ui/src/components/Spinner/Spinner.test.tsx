import React from 'react';
import { shallow } from 'enzyme';

import Spinner from './Spinner';

describe('Spinner', () => {
    it('Renders without crashing', () => {
        const element = shallow(<Spinner />);
        expect(element).toMatchSnapshot();
    });
});
