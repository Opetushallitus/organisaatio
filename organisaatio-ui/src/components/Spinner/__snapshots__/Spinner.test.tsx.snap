// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner Renders without crashing 1`] = `<div />`;
